docs/en/query_language/functions/array_functions.md

```diff
@@ -458,6 +458,85 @@ SELECT arraySlice([1, 2, NULL, 4, 5], 2, 3) AS res
 
 Array elements set to `NULL` are handled as normal values.
 
+## arraySort(arr)
+
+Returns an array as result of sorting the elements of `arr` in ascending order.  
+
+The [Schwartzian transform](https://en.wikipedia.org/wiki/Schwartzian_transform) is used to improve sorting efficiency.
+
+Example 1
+
+``` sql
+SELECT arraySort([1, 3, 3, 0])
+```
+```
+┌─arraySort([1, 3, 3, 0])─┐
+│ [0,1,3,3]               │
+└─────────────────────────┘
+```
+
+Example 2
+
+``` sql
+SELECT arraySort(['hello', 'world', '!'])
+```
+```
+┌─arraySort(['hello', 'world', '!'])─┐
+│ ['!','hello','world']              │
+└────────────────────────────────────┘
+```
+
+Note that NULLs and NaNs go last (NaNs go before NULLs). For example:
```

Comment:
* Sounds weird, could be "NULLs go last, NaNs go right before NULLs"
* Positive and negative infinity are probably also worth mentioning